jax/interpreters/xla.py

```diff
@@ -93,6 +94,15 @@ def _make_array_shape(a):
   else:
     return (xc.Shape.array_shape(a.dtype, a.shape),)
 
+def _get_source_file(frame: source_info_util.Frame):
```

Comment:
Perhaps `_get_canonical_source_path` ?

Reply:
I did `_get_canonical_source_file` to match the existing terminology. It's more accurately a path, but it's called a source_file.